patches/@parcel+transformer-postcss+2.0.0-nightly.951.patch

```diff
@@ -0,0 +1,77 @@
+diff --git a/node_modules/@parcel/transformer-postcss/lib/PostCSSTransformer.js b/node_modules/@parcel/transformer-postcss/lib/PostCSSTransformer.js
```

Comment:
This patch is necessary because we currently require a very old version of PostCSS which is no longer supported by Parcel.